source/gameengine/Ketsji/KX_MeshProxy.cpp

```diff
@@ -115,6 +117,29 @@ STR_String& KX_MeshProxy::GetName()
 }
 
 // stuff for python integration
+static int kx_mesh_proxy_get_polygons_size_cb(void *self_v)
+{
+	return ((KX_MeshProxy *)self_v)->GetMesh()->NumPolygons();
+}
+
+static PyObject *kx_mesh_proxy_get_polygons_item_cb(void *self_v, int index)
+{
+	KX_MeshProxy *self = static_cast<KX_MeshProxy *>(self_v);
+	RAS_Polygon *polygon = self->GetMesh()->GetPolygon(index);
+	PyObject *polyob = (new KX_PolyProxy(self, polygon))->NewProxy(true);
+	return polyob;
+}
+
+PyObject *KX_MeshProxy::pyattr_get_polygons(void *self_v, const KX_PYATTRIBUTE_DEF *attrdef)
```

Comment:
I would prefer move these function after `KX_MeshProxy::pyattr_get_numMaterials` or `KX_MeshProxy::pyattr_get_numPolygons` to avoid mix python attributes and functions.